docs/source-pytorch/common/lightning_module.rst

```diff
@@ -18,12 +18,6 @@ A :class:`~LightningModule` organizes your PyTorch code into 6 sections:
 
 |
 
-.. raw:: html
```

Comment:
Unfortunately, just removing it won't be enough. The section here would have to be rewritten. See how the text continues off of the animation.

Reply:
I tried rewriting it. Take a look! Thanks

Reply:
LGTM. I would just wait a moment for Eden's reply. She might have the old keynote and if not, fine with me get this removed.